arkouda-docker/README.md

```diff
@@ -72,16 +83,18 @@ The arkouda-udp-server image delivers a GASNET udp configuration that enables de
 ## Building arkouda-udp-server
 
 ```
+export CHAPEL_UDP_IMAGE=hokiegeek2/chapel-gasnet-udp:1.27.0
 export ARKOUDA_DISTRO_NAME=v2022.10.13
 export ARKOUDA_DOWNLOAD_URL=https://github.com/Bears-R-Us/arkouda/archive/refs/tags/v2022.10.13.zip
 export ARKOUDA_BRANCH_NAME=2022.10.13
 export ARKOUDA_INTEGRATION_DOWNLOAD_URL=https://github.com/Bears-R-Us/arkouda-contrib/archive/refs/heads/main.zip
 export ARKOUDA_INTEGRATION_DISTRO_NAME=main
 
-docker build --build-arg ARKOUDA_DISTRO_NAME=$ARKOUDA_DISTRO_NAME \
+docker build --build-arg CHAPEL_UDP_IMAGE=$CHAPEL_UDP_IMAGE \
+             --build-arg ARKOUDA_DISTRO_NAME=$ARKOUDA_DISTRO_NAME \
              --build-arg ARKOUDA_DOWNLOAD_URL=$ARKOUDA_DOWNLOAD_URL \
              --build-arg ARKOUDA_INTEGRATION_DISTRO_NAME=$ARKOUDA_INTEGRATION_DISTRO_NAME \
              --build-arg ARKOUDA_BRANCH_NAME=$ARKOUDA_BRANCH_NAME \
              --build-arg ARKOUDA_INTEGRATION_DOWNLOAD_URL=$ARKOUDA_INTEGRATION_DOWNLOAD_URL \
-             -f Dockerfile -t hokiegeek2/arkouda-udp-server:$ARKOUDA_DISTRO_NAME .
+             -f arkouda-udp-server -t hokiegeek2/arkouda-udp-server:$ARKOUDA_DISTRO_NAME .
```

Comment:
we are still using `hokiegeek2` here

Reply:
@pierce314159 beat me to this, but I had the same thought.